environments/production/public.yml

```diff
@@ -158,6 +157,8 @@ localsettings:
         - 'ews-ghana'
         - 'ils-gateway'
         - 'ils-gateway-train'
+  USER_REPORTING_METADATA_BATCH_ENABLED: True
```

Comment:
Should also consider reviewing/updating `USER_REPORTING_METADATA_BATCH_SCHEDULE`